scene/resources/mesh.cpp

```diff
@@ -480,9 +480,6 @@ void Mesh::_bind_methods() {
 	BIND_ENUM_CONSTANT(PRIMITIVE_TRIANGLES);
 	BIND_ENUM_CONSTANT(PRIMITIVE_TRIANGLE_STRIP);
 
-	BIND_ENUM_CONSTANT(BLEND_SHAPE_MODE_NORMALIZED);
-	BIND_ENUM_CONSTANT(BLEND_SHAPE_MODE_RELATIVE);
-
 	BIND_ENUM_CONSTANT(ARRAY_FORMAT_VERTEX);
```

Comment:
All the other enum bindings here are also redundant (at least on current `master`).